generate/schemesgen/main.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2017, Shreyas Khare <skhare@rapid7.com>
+// See LICENSE for licensing information
+
+package main
```

Comment:
This program needs to be in a `go:generate` line in `xurls.go`, so that running `go generate` updates all generated files.

Also, is the generated file already in the `gofmt` format? It should be.

Reply:
As you can obviously tell, this is my first crack at any type of Go code (have the Go tutorial open in another tab). Please feel free to bash my coding style and help me fix it, the right way.

Reply:
They're not used by the compiler - they're just comments - but the `go generate` tool does use these comments.